### PR TITLE
Frontend v. 1.3

### PR DESCRIPTION
**Pull request by Fisz-git**
Added .json import (experimental)
Path for fetch to be changed accordingly.

### CODE REVIEW

File: src/lottery.html
Comment (theSusurrus) [inline src/lottery.html:1]: Please remove

File: host/lottery.html [hunk 1]
```diff
@@ -99,33 +104,48 @@
...
+//Tutaj będzie pobieranie pliku z listą graczy z serwera
+async function getNames () {
+    const response = await fetch('./names.json');
+    const importLista = await response.json();
+    listaGraczy = importLista;
+    displayLosowanie = listaGraczy.join('\n');
+    document.getElementById("lista").value = displayLosowanie;
+    document.getElementById("pozostalo").value = "Pozostało " + importLista.length + " graczy";
+    document.getElementById("losowany").style.backgroundColor="#7a1414"
+    gracz = "Import poprawny - liczba graczy: " + importLista.length;
```
Comment (theSusurrus) [inline host/lottery.html]: This prints on red background, which is confusing, since everything went OK. Please change the color to something more natural.

**Review by theSusurrus — CHANGES_REQUESTED** [review]

File: src/lottery.html
Comment (Fisz-git) [reply src/lottery.html:1]: Deleted in 1.4.1

File: host/lottery.html (hunk 1 above)
Comment (Fisz-git) [reply host/lottery.html]: Changed to sky blue in 1.4.1

**Review by theSusurrus — APPROVED** [review]

**Pull request merged**